src/nodes/projectiles/alien_laser.lua

```diff
@@ -35,6 +35,9 @@ return{
       node:hurt(projectile.damage, projectile.special_damage, 0)
       projectile:die()
     end
+    if node.isBuilding then
```

Comment:
This should probably be `if node.isBuilding and node.burn then` just incase we eventually add building nodes that can't be burned down, I know we probably won't but its nice to have that covered.